scripts/ci-e2e.sh

```diff
@@ -45,8 +45,8 @@ export IMAGE_OS=${IMAGE_OS}
 export FORCE_REPO_UPDATE="false"
 export USE_IRSO="${USE_IRSO:-false}"
 EOF
-# if running a scalability test skip apply bmhs in dev-env and run fakeIPA
-if [[ ${GINKGO_FOCUS:-} == "clusterctl-upgrade" ]]; then
+# if running basic integration or clusterctl-upgrade test skip apply bmhs in dev-env
```

Comment:
I'd love to see this comple if switched to `case` in a follow-up for readability.

Reply:
Done